resources/lib/iptvmanager.py

```diff
@@ -48,7 +48,7 @@ def send_channels():  # pylint: disable=no-method-argument
                 stream='plugin://plugin.video.vrt.nu/play/id/{live_stream_id}'.format(**channel),
             )
             if channel.get('has_tvguide'):
-                item.update(dict(vod='plugin://plugin.video.vrt.nu/play/airdate/{name}/{{date}}'.format(**channel)))
+                item.update(dict(vod='plugin://plugin.video.vrt.nu/play/airdate/{name}/{{start}}/{{stop}}'.format(**channel)))
```

Comment:
Can we change this to `{{start}}/{{duration}}` and calculate the endpoint here? I agree that `date` isn't the best name for this field.

Reply:
Currently, the VRT NU airdate interface uses two ISO 8601 timestamps. I would rather not change the current interface unless you have a very good reason.
In addition, duration is in seconds and I think mixing various time formats in one url is confusing.

Reply:
This is fine, I can support both.